Source/CarthageKit/VersionFile.swift

```diff
@@ -8,10 +8,12 @@ struct CachedFramework: Codable {
 	enum CodingKeys: String, CodingKey {
 		case name = "name"
 		case hash = "hash"
+		case swiftVersion = "swiftVersion"
```

Comment:
not sure this is the most appropriate name, maybe it should be toolchain version? @ikesyo

Reply:
It should be `toolchainVersion` rather than `swiftVersion`?

Reply:
Lets see what others think